### PR TITLE
adding the feature to highlight the selected JOB node and highlighting the right clicked job or job file

### PR DESCRIPTION
**Pull request by AdarshdeepCheema**
adding the feature to highlight the selected JOB node only and highlighting the right clicked job or job file
Currently when we select a job then it also highlights the job files under it. and when we right click the job File then ti does not highlight it.

Signed-off-by: Adarshdeep Cheema <adarshdeep.cheema@ibm.com>

This PR addresses Issue: [*Link to Github issue within https://github.com/zowe/zlux/issues* if any]

## PR Type
- [ ] Bug fix
- [X] Feature
- [ ] Other (Please indicate)

## PR Checklist
- [ ] PR completes `npm run preCommit` without error
- [ ] Relevant Test cases have been added (Unit and or FVT)
- [ ] Relevant update to CHANGELOG.md
- [ ] PR from forked repo? Ensure Allow edits by maintaners is set.

### CODE REVIEW

**Comment by AdarshdeepCheema** [comment]
Screen Shot 2022-02-09 at 12 39 37 PM

File: WebContent/js/containers/ContentViewer.js [hunk 1]
```diff
@@ -135,7 +135,7 @@ export class ContentViewer extends React.Component {
         }
     }
 
-    handleCloseAllExceptTabs(index) {
+    handleCloseOtherTabs(index) {
```
Comment (DivergentEuropeans) [inline WebContent/js/containers/ContentViewer.js:138]: I don't think these Close All changes were intended

**Review by DivergentEuropeans — CHANGES_REQUESTED** [review]
Hey, I noticed a few small UX areas for polish:

image

When we right click a node, the other node doesn't get deselected. As I understand, our current UX is: 

highlighting - this node is ready to be selected (by keyboard or mouse etc)
border outline - this node is selected

so when we right click a different node, the context menu should align with the border outline nodes. And currently there's 2 outlined instead of 1

File: WebContent/js/containers/ContentViewer.js (hunk 1 above)
Comment (AdarshdeepCheema) [reply WebContent/js/containers/ContentViewer.js:138]: I just changed the function names and Context Menu option names to more user friendly

**Review by DivergentEuropeans — APPROVED** [review]
Changes addressed

**Pull request merged**